rasa_sdk/forms.py

```diff
@@ -689,3 +689,76 @@ def _get_entity_type_of_slot_to_fill(
                 return None
 
         return entity_type
+
+
+class FormValidationAction(Action):
+    """An action that validates if every extracted slot is valid."""
+
+    def name(self) -> Text:
+        """Unique identifier of this action.
+
+        Returns:
+            Name of the action.
+        """
+
+        raise NotImplementedError("An action must implement a name")
```

Comment:
we don't have to override this abstract method with another abstract method. You might need a `class FormValidationAction(action, ABC)` though to mark `FormValidationAction` as abstract